packages/react-layouts/src/Modal/index.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright (c) Quid, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @flow
+
+import * as React from 'react';
+import ReactModal from 'react-modal';
+import styled from '@emotion/styled/macro';
+import Centerer from './Centerer';
+import { withFallback as wf } from '@quid/theme';
+import Color from 'color';
+import { ClassNames } from '@emotion/core';
+import Header from './Header';
+import { ACTION } from './importanceTypes';
+import type { Importance } from './importanceTypes';
+import ActionBar from './ActionBar';
+import css from '@emotion/css/macro';
+
+type Props = {
+  title?: React.Node,
+  alt?: string, // specify if `title` is not a string
+  icon?: string,
+  noPadding?: boolean,
+  importance?: Importance,
+  ignoreHeightLimits?: boolean,
+  centerVertically?: boolean,
+  action?: React.Node,
+  renderActionLeft?: string => React.Node,
+  renderActionRight?: string => React.Node,
+  children: React.Node,
+  overlayClassName?: string,
+  styles?: Object,
```

Comment:
This type is unused

Reply:
Also `className` below, maybe we should double check them all to make sure we don't define un-needed types

Reply:
Removed both